packages/react-components/__tests__/components/Accordion.test.tsx

```diff
@@ -0,0 +1,113 @@
+import Accordion from "../../src/components/Accordion";
```

Comment:
All files should have adobe copyright headers.

Reply:
Implemented!